LevelUp.Pos.ProposedOrders/LevelUp.Pos.ProposedOrders.csproj

```diff
@@ -3,6 +3,8 @@
     <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
     <PackageId>LevelUp.Pos.ProposedOrderCalculator</PackageId>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageOutputPath>$(SolutionDir)\nuget</PackageOutputPath>
```

Comment:
Can we keep this the default and just change the artifacts folder in `appveyor.yml`?

Reply:
I think it makes more sense to keep using the nuget directory. Makes it easier to use it as a local package source, otherwise you have to individually add Debug and Release folders for each target framework as package sources.

Reply:
We should only add the release directory, something like this: https://github.com/TheLevelUp/pos-ordering-lib/blob/31266dd1ada83cf623ffa855bc5fc6eab3b985fc/appveyor.yml

Reply:
We decided we'd be using \artifacts for everything, so all final artifact output should go there.

Reply:
> We should only add the release directory, something like this: https://github.com/TheLevelUp/pos-ordering-lib/blob/31266dd1ada83cf623ffa855bc5fc6eab3b985fc/appveyor.yml

The AppVeyor build will only build Release mode anyway, and will start with a clean artifacts directory, so there's no need to additionally specify. On local machine I would like to have debug builds output packages to the same directory for the purpose of local nuget sources.